jaeger_client/TUDPTransport.py

```diff
@@ -26,30 +27,31 @@ class TUDPTransport(TTransportBase, object):
     TUDPTransport implements just enough of the tornado transport interface
     to work for blindly sending UDP packets.
     """
-    def __init__(self, host, port, blocking=False):
+    def __init__(self, host: str, port: int, blocking: bool = False) -> None:
         self.transport_host = host
         self.transport_port = port
-        self.transport_sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-        if blocking:
-            blocking = 1
-        else:
-            blocking = 0
```

Comment:
Is this safe to remove? Does transport_sock.setblocking accept booleans and ints?

I'd be more comfortable with keeping the existing logic, just folding it into an expression
```
self.transport_sock.setblocking(1 if blocking else 0)
```

Reply:
So according to the docs, it never accepted ints: https://docs.python.org/3/library/socket.html#socket.socket.setblocking